Feature/dtm 10.15.0.0 release notes (#44)
SoftwareAG/c8y-releasenotes: content/release-10-15-0/whatsnew-10-15-0.md

In an effort to provide you a more consistent experience across the platform the **All devices** page has been re-shaped. It is now implemented in the data grid fashion you already are familiar with from **Subassets** and **Search** pages. This offers improved appearence in various viewport sizes, easier filtering and sorting, pagination option (only for users with global inventory access) and sets solid ground for further improvements.

![All devices](/images/release-notes/devmgmt-devices-alldevices.png)

### Digital Twin Manager

Cumulocity IoT Digital Twin Manager is a layered product hosted on Cumulocity platform. The Digital Twin Manager enables you to create and manage assets around your physical connected devices in Cumulocity IoT. Assets can be used to structure and describe devices in logical hierarchies as encountered in the real world environment.

> The Digital Twin Manager consists of several components, which support you to create your assets and asset hierarchies:

Reviewer: ```suggestion
The DTM consists of several components, which allow you to create your assets and asset hierarchies:
```

>     •	Property library - Create your own custom properties (simple or complex) to define how data should be stored on an asset

Reviewer: ```suggestion
    •	**Property library** - Create your own custom properties (simple or complex) to define how data should be stored on an asset
```

    •	**Asset types** - Define the asset types to describe the blueprint of your assets
    •	**Localization** - Dynamically translate the asset type and custom property descriptions to multiple languages
    •	**Assets navigator** - Manage your asset hierarchies

> You can access the DTM from the Cumulocity IoT Application Switcher:

Reviewer 2: ```suggestion
You can access the DTM from the Cumulocity IoT application switcher:
```

> ![App switcher](/images/dtm/app-switcher-dtm.png)

Reviewer: ```suggestion
![Application Switcher](/images/dtm/app-switcher-dtm.png)
```

> ![App switcher](/images/dtm/app-switcher-dtm.png)

Reviewer 2: This image is not of much value. I would better show the DTM here.

Author: Is it DTM landing screen?

Reviewer 2: Yes, I think that is better. Showing the app switcher does not make much sense.

Author: Then I will add the DTM landing page screenshot

Author: I have updated the image and committed. Please review and let me know if anything needs to be modified

> ![Application Switcher](/images/dtm/app-switcher-dtm.png)

Reviewer 2: ```suggestion
![Application switcher](/images/dtm/app-switcher-dtm.png)
```

> See the [Cumulocity IoT DTM Guide](/guides/dtm/#overview) for the complete documentation.

Reviewer 2: ```suggestion
See the [Cumulocity IoT DTM guide](/guides/dtm/#overview) for the complete documentation.
```

> See the [Cumulocity IoT DTM guide](/guides/dtm/#overview) for the complete documentation.

Reviewer 2: ```suggestion
For detailed information, see the [Cumulocity IoT DTM guide](/guides/dtm/#overview).
```

**Asset Types**

Asset Types serve as a blueprint for your assets and define how they are structured. An Asset Type defines what custom properties an asset should have. Additionally, you can specify relationships between multiple Asset Types, which will be respected when you create your actual asset hierarchies. Once the Asset Type is created, it can be used to create various assets.

**Assets Navigator**

The Assets Navigator is the starting point to create your asset hierarchies based on the asset types and custom properties, you have defined previously. Once you have created your asset hierarchies you can use the Assets Navigator to view and manage them.

**Localization**

You can use the Localization component to define translations for your asset types and custom properties. These translations can be provided for all supported languages in Cumulocity IoT. This component requires to have the feature-branding feature subscribed for your Cumulocity tenant.

> ### Installation

Reviewer 2: Needs one more # to have the correct heading level

Author: Added one more #

> See the [Cumulocity IoT DTM Installation & operations guide](<<TBD>>) for the complete documentation.

Reviewer 2: ```suggestion
For information on installation and operations, see the *Digital Twin Manager -  Installation & operations guide*.
```

Reviewer 2: We can not link to the installation documentation as these are only available as PDF.

Author: Do we need to mention all the installation steps which we have documented in Ops guide?
Re writing the same content is fine?
